libnd4j/tests_cpu/run_tests.sh

```diff
@@ -34,4 +35,12 @@ parse_commandline ()
 
 parse_commandline "$@"
 
-cmake -G "Unix Makefiles" -D_RELEASE=${IS_RELEASE} && make -j4 && layers_tests/runtests --gtest_output="xml:../target/surefire-reports/TEST-results.xml"
+cmake -G "Unix Makefiles" -D_RELEASE=${IS_RELEASE} && make -j4
+
+if [[ "$OSTYPE" == "linux-gnu" && "$OSARCH" == "x86_64" ]]; then
+    # sudo is used as workaround for LeakSanitizer that requires root permissions
+    sudo bash -c './layers_tests/runtests --gtest_output="xml:../target/surefire-reports/TEST-results.xml"'
+    sudo chown -R "${USER:-jenkins}":"${USER:-jenkins}" ../target
```

Comment:
I don't think it's a good idea to hard code values for the CI server here. That won't work outside Jenkins.

Reply:
If you mean `jenkins`, then it's not hard coded, because first we check `USER` variable.
But, agree that it's not the best way, at the same time this scripts mostly used by CI, as far as I know.

Reply:
Well, as long as we can easily run it outside Jenkins, I'm OK with it... Did you test it outside Jenkins?

Reply:
@saudet right, tested it locally, works fine.